mappings/net/minecraft/util/Identifier.mapping

```diff
@@ -1,30 +1,31 @@
 CLASS qs net/minecraft/util/Identifier
 	CLASS qs$a Serializer
-		METHOD deserialize (Lcom/google/gson/JsonElement;Ljava/lang/reflect/Type;Lcom/google/gson/JsonDeserializationContext;)Ljava/lang/Object;
-			ARG 1 functionJson
-			ARG 2 unused
-			ARG 3 context
 	FIELD a namespace Ljava/lang/String;
 	FIELD b path Ljava/lang/String;
-	FIELD c EXCEPTION_INVALID Lcom/mojang/brigadier/exceptions/SimpleCommandExceptionType;
-	METHOD <init> (Ljava/lang/String;)V
+	FIELD c COMMAND_EXCEPTION Lcom/mojang/brigadier/exceptions/SimpleCommandExceptionType;
+	METHOD <init> ([Ljava/lang/String;)V
```

Comment:
Is the parameter still `id` after the type changed to Array?

Reply:
It's an array of components that form the `id`

Reply:
`sanitizedParts`

Reply:
The parts are not yet sanitized when passed into the constructor, it is merely a split string.